LINT ERROR(make test): File is not `goimports`-ed, pkg/minikube/registry/drvs/{vmwarefusion, hyperkit}/driver.go

<!-- Please use this template while reporting an issue, providing as much information as possible. Failure to do so may result in a delayed response. Thank you! -->

**The exact command to reproduce the issue**:

when the following command exec, and we will get test fail due to lint error.

```
make test
= make lint =============================================================
pkg/minikube/registry/drvs/vmwarefusion/driver.go:26: File is not `goimports`-ed (goimports)
	cfg "k8s.io/minikube/pkg/minikube/config"
pkg/minikube/registry/drvs/hyperkit/driver.go:27: File is not `goimports`-ed (goimports)
	cfg "k8s.io/minikube/pkg/minikube/config"
make[1]: *** [lint-ci] Error 1
```

**The full output of the command that failed**:<details>

```
$ make test
./test.sh
= make lint =============================================================
pkg/minikube/registry/drvs/vmwarefusion/driver.go:26: File is not `goimports`-ed (goimports)
	cfg "k8s.io/minikube/pkg/minikube/config"
pkg/minikube/registry/drvs/hyperkit/driver.go:27: File is not `goimports`-ed (goimports)
	cfg "k8s.io/minikube/pkg/minikube/config"
make[1]: *** [lint-ci] Error 1
= go mod ================================================================
ok
= boilerplate ===========================================================
ok
= schema_check ==========================================================
ok
= go test ===============================================================
ok  	k8s.io/minikube/cmd/minikube/cmd	0.204s	coverage: 18.4% of statements
ok  	k8s.io/minikube/cmd/minikube/cmd/config	0.061s	coverage: 19.8% of statements
ok  	k8s.io/minikube/pkg/drivers	0.033s	coverage: 19.6% of statements
ok  	k8s.io/minikube/pkg/drivers/hyperkit	0.029s	coverage: 13.8% of statements
ok  	k8s.io/minikube/pkg/minikube/assets	0.041s	coverage: 63.1% of statements
ok  	k8s.io/minikube/pkg/minikube/bootstrapper	1.992s	coverage: 73.3% of statements
ok  	k8s.io/minikube/pkg/minikube/bootstrapper/kubeadm	0.084s	coverage: 30.9% of statements
ok  	k8s.io/minikube/pkg/minikube/cluster	0.420s	coverage: 56.8% of statements
ok  	k8s.io/minikube/pkg/minikube/config	0.043s	coverage: 76.2% of statements
ok  	k8s.io/minikube/pkg/minikube/cruntime	0.022s	coverage: 64.0% of statements
ok  	k8s.io/minikube/pkg/minikube/extract	0.021s	coverage: 56.9% of statements
ok  	k8s.io/minikube/pkg/minikube/kubeconfig	0.061s	coverage: 75.9% of statements
ok  	k8s.io/minikube/pkg/minikube/logs	0.039s	coverage: 1.4% of statements
ok  	k8s.io/minikube/pkg/minikube/machine	7.733s	coverage: 23.1% of statements
ok  	k8s.io/minikube/pkg/minikube/notify	0.111s	coverage: 92.9% of statements
ok  	k8s.io/minikube/pkg/minikube/out	0.029s	coverage: 80.0% of statements
ok  	k8s.io/minikube/pkg/minikube/problem	0.018s	coverage: 100.0% of statements
ok  	k8s.io/minikube/pkg/minikube/proxy	0.026s	coverage: 100.0% of statements
ok  	k8s.io/minikube/pkg/minikube/registry	0.029s	coverage: 100.0% of statements
ok  	k8s.io/minikube/pkg/minikube/service	2.999s	coverage: 85.2% of statements
ok  	k8s.io/minikube/pkg/minikube/sshutil	1.003s	coverage: 100.0% of statements
ok  	k8s.io/minikube/pkg/minikube/storageclass	0.028s	coverage: 97.2% of statements
ok  	k8s.io/minikube/pkg/minikube/translate	0.010s	coverage: 11.9% of statements
ok  	k8s.io/minikube/pkg/minikube/tunnel	1.819s	coverage: 59.1% of statements
ok  	k8s.io/minikube/pkg/util	0.592s	coverage: 62.0% of statements
ok  	k8s.io/minikube/pkg/util/lock	0.011s	coverage: 59.1% of statements
ok  	k8s.io/minikube/pkg/util/retry	0.010s	coverage: 0.0% of statements
ok
make: *** [test] Error 4
```

</details>

**The output of the `minikube logs` command**:<details>

nothing

</details>

**The operating system version**:

 * macOS 10.14.5
 * minikube version: v1.5.0-beta.0
commit: 5ba4a7d66ca47a57d3b06f75e21e8e11134df7f0

I'll send PR to fix this lint error.

--- a/pkg/minikube/registry/drvs/hyperkit/driver.go
+++ b/pkg/minikube/registry/drvs/hyperkit/driver.go
@@ -25,9 +25,9 @@ import (
 	"github.com/pborman/uuid"
 	"k8s.io/minikube/pkg/drivers/hyperkit"
 	cfg "k8s.io/minikube/pkg/minikube/config"
+	"k8s.io/minikube/pkg/minikube/driver"
 	"k8s.io/minikube/pkg/minikube/localpath"
 	"k8s.io/minikube/pkg/minikube/registry"
-	"k8s.io/minikube/pkg/minikube/driver"
 )
 
 func init() {

--- a/pkg/minikube/registry/drvs/vmwarefusion/driver.go
+++ b/pkg/minikube/registry/drvs/vmwarefusion/driver.go
@@ -24,10 +24,9 @@ import (
 	"github.com/docker/machine/drivers/vmwarefusion"
 	"github.com/docker/machine/libmachine/drivers"
 	cfg "k8s.io/minikube/pkg/minikube/config"
+	"k8s.io/minikube/pkg/minikube/driver"
 	"k8s.io/minikube/pkg/minikube/localpath"
 	"k8s.io/minikube/pkg/minikube/registry"
-	"k8s.io/minikube/pkg/minikube/driver"
-
 )
 
 func init() {

